--- Samples/Chat Example/Chat Example Server (old).cpp	2011-02-10 22:29:59.000000000 -0600
+++ Samples/Chat Example/Chat Example Server.cpp	2011-02-10 22:36:48.805600990 -0600
@@ -17,9 +17,9 @@
 #include <cstdio>
 #include <cstring>
 #include <stdlib.h>
-#ifdef _WIN32
+//#ifdef _WIN32  // Kbhit.h is needed on Linux. Maybe this should be "ifndef"?  -- Dennis
 #include "Kbhit.h"
-#endif
+//#endif
 #include <stdio.h>
 #include <string.h>
 #include "Gets.h"
--- Samples/Dropped Connection Test/DroppedConnectionTest (old).cpp	2011-02-10 22:32:04.000000000 -0600
+++ Samples/Dropped Connection Test/DroppedConnectionTest.cpp	2011-02-10 22:38:42.255601004 -0600
@@ -82,12 +82,13 @@
 		// User input
 		if (kbhit())
 		{
-#ifndef _WIN32
-			Gets(buff,sizeof(buff));
-			ch=buff[0];
-#else
+// No Gets on Ubuntu. Just use the getch() from Kbhit.h.  -- Dennis
+//#ifndef _WIN32
+//			Gets(buff,sizeof(buff));
+//			ch=buff[0];
+//#else
 			ch=getch();
-#endif			
+//#endif			
 
 			if (ch=='d' || ch=='D')
 			{
